Crashlytics/Crashlytics/FIRCrashlytics.m

```diff
@@ -400,11 +406,13 @@ - (void)recordError:(NSError *)error {
 }
 
 - (void)recordError:(NSError *)error userInfo:(NSDictionary<NSString *, id> *)userInfo {
-  FIRCLSUserLoggingRecordError(error, userInfo);
+  NSString *rolloutsInfo = [_remoteConfigManager getRolloutAssignmentsEncodedJsonString];
+  FIRCLSUserLoggingRecordError(error, userInfo, rolloutsInfo);
 }
 
 - (void)recordExceptionModel:(FIRExceptionModel *)exceptionModel {
-  FIRCLSExceptionRecordModel(exceptionModel);
+  NSString *rolloutsInfo = [_remoteConfigManager getRolloutAssignmentsEncodedJsonString];
+  FIRCLSExceptionRecordModel(exceptionModel, rolloutsInfo);
```

Comment:
Same here about the -JSON suffix

Reply:
Changed to `rolloutsInfoJSON` everywhere!